src/components/Graph/Graph.tsx

```diff
@@ -174,70 +168,21 @@ const Graph: FC<GraphProps> = ({
     : [];
 
   return (
-    <div
-      ref={graphElement}
-      style={{
-        height: interactive ? maxHeight + 150 : 'inherit',
-        position: 'relative',
-        overflow: 'auto',
-        marginRight: '5px'
-      }}
-    >
-      {nodeCoordinates !== undefined
-        ? Object.keys(nodeCoordinates).map(key => {
-            const docResource = documentation.find((doc): doc is DocumentationResource => {
-              return typeof doc !== 'string' && doc.state === key;
-            });
-            const isCurrentNode = (): boolean => {
-              return path[path.length - 1] === key;
-            };
-            const onClickHandler = interactive ? (): void => setExpanded(key) : undefined;
-            return (
-              <Node
-                key={key}
-                documentation={docResource}
-                ref={(node: HTMLDivElement): void => {
-                  nodeRefs.current[key] = node;
-                }}
-                pathwayState={pathway.states[key]}
-                isOnPatientPath={path.includes(key)}
-                isCurrentNode={isCurrentNode()}
-                xCoordinate={nodeCoordinates[key].x + windowWidth / 2}
-                yCoordinate={nodeCoordinates[key].y}
-                expanded={expanded[key]}
-                onClickHandler={onClickHandler}
-              />
-            );
-          })
-        : []}
-      <svg
-        xmlns="http://www.w3.org/2000/svg"
-        style={{
-          // Adding 5 pixels to maxWidth so that the rightmost edge label is not cut off
-          width: maxWidth + 5,
-          height: maxHeight,
-          zIndex: 1,
-          top: 0,
-          left: 0,
-          overflow: 'visible'
-        }}
-      >
-        {edges !== undefined
-          ? Object.keys(edges).map(edgeName => {
-              const edge = edges[edgeName];
-              return (
-                <Arrow
-                  key={edgeName}
-                  edge={edge}
-                  edgeName={edgeName}
-                  isOnPatientPath={isEdgeOnPatientPath(path, edge)}
-                  widthOffset={windowWidth / 2}
-                />
-              );
-            })
-          : []}
-      </svg>
-    </div>
+    <GraphRender
+      graphElement={graphElement}
+      interactive={interactive}
+      maxHeight={maxHeight}
+      nodeCoordinates={nodeCoordinates}
+      documentation={documentation}
+      edges={edges}
+      path={path}
+      nodeRefs={nodeRefs}
+      windowWidth={windowWidth}
+      maxWidth={maxWidth}
+      pathway={pathway}
+      expanded={expanded}
+      setExpanded={setExpanded}
+    ></GraphRender>
```

Comment:
Since there are no children you should use a self-closing tag
```suggestion
    />
```